Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.21 as builder
+FROM golang:1.21 AS builder
```

Comment:
on `make docker-build` I saw a warning about inconsistent casing in the dockerfile, just a drive-by